js/kucoin.js

```diff
@@ -2337,5 +2337,6 @@ module.exports = class kucoin extends Exchange {
         const message = this.safeString (response, 'msg', '');
         this.throwExactlyMatchedException (this.exceptions['exact'], message, this.id + ' ' + message);
         this.throwExactlyMatchedException (this.exceptions['exact'], errorCode, this.id + ' ' + message);
+        this.throwBroadlyMatchedException (this.exceptions['broad'], body, this.id + ' ' + message);
```

Comment:
We don't have to glue `this.id + ' ' + message` three times here, instead we should do this:

```JavaScript
        const feedback = this.id + ' ' + body;
        this.throwExactlyMatchedException (this.exceptions['exact'], message, feedback);
        this.throwExactlyMatchedException (this.exceptions['exact'], errorCode, feedback);
        this.throwBroadlyMatchedException (this.exceptions['broad'], body, feedback);
```